File: tks/RestAdapters/src/main/kotlin/dto/LibraryItemDto.java
package dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class LibraryItemDto {
    private String accessionNumber;
    private String title;
    private String author;
    private String publisher;
    private String type;

    public String getType() {
        return type;
    }
}

File: tks/RestAdapters/src/main/kotlin/rest/api/dto/LibraryItemDto.kt
package rest.api.dto

class LibraryItemDto() {
    var accessionNumber: String? = null
    var title: String? = null
    var author: String? = null
    var publisher: String? = null
    var type: String? = null

    constructor(accessionNumber: String?, title: String?, author: String?, publisher: String?, type: String?) : this() {
        this.accessionNumber = accessionNumber
        this.title = title
        this.author = author
        this.publisher = publisher
        this.type = type
    }
}